src/content/docs/en/guides/cms/hashnode.mdx

```diff
@@ -0,0 +1,352 @@
+---
+title: Hashnode & Astro
+description: Add content to your Astro project using Hashnode as a CMS
+type: cms
+stub: false
+service: Hashnode
+i18nReady: true
+---
+
+import { FileTree } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Hashnode](https://hashnode.com/) is a hosted CMS that allows you to create a blog or publication.
+
+## Integrating with Astro
+The [Hashnode Public API](https://apidocs.hashnode.com/) is a GraphQL API that allows you to interact with Hashnode. This guide uses [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), a minimal GraphQL client that works well with Astro, to bring your Hashnode data into your Astro project.
+
+### Prerequisites
+To get started you will need to have the following: 
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **A Hashnode site** - You can create free personal site by visiting [Hashnode](https://hashnode.com/).
+
+### Installing dependencies
+
+Install the `graphql-request` package using the package manager of your choice:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install graphql-request
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add graphql-request
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add graphql-request
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Fetching data
+
+Can we have a minimal example here?
```

Comment:
The current recipe is fairly minimal, the steps in line 62 could be trimmed down and put here but its repeated information which is 😬

Reply:
Sarah put that in... not sure what example she was looking for... I honestly thought the guide was fairly minimal as is, as most the code was directly pulled and simplified a bit from my `astro-hashnode` integration :sweat_smile:

Reply:
SARAH WILL TAKE THE BLAME, lol

Lemme give a quick once over so we can get this puppy merged!